### ISSUE TITLE
Some important feedforward models to be supported at first stage

### ISSUE THREAD

**Issue by kuke** [PaddlePaddle/Paddle2ONNX#24]
We are going to support  the conversion of models as follows at first stage. They all can be found in our models bank, some maybe only need to be verified after #19 merged, and some lack of necessary operators. 

- [x]  [recognize_digits](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/book/test_recognize_digits.py) ( #19 @kuke)
- [x]  [ResNet50](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/book/test_image_classification.py#L27)  (#30 #32  @kuke)
- [x]  [VGG16](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/book/test_image_classification.py#L69) (#30 #32  @kuke)
- [x]  [MobileNet](https://github.com/PaddlePaddle/models/blob/develop/fluid/image_classification/mobilenet.py) (#30 #33  @kuke)
- [x]  [SE-ResNeXt](https://github.com/PaddlePaddle/models/blob/develop/fluid/image_classification/se_resnext.py) (#30 #33  @kuke)
- [x]  Inception-v4 (first add the Fluid config in [models/fluid/image_classification](https://github.com/PaddlePaddle/models/tree/develop/fluid/image_classification)) ([#900](https://github.com/PaddlePaddle/models/pull/900) [#49](https://github.com/PaddlePaddle/paddle-onnx/pull/49) @kuke)

Even all the models above are supported, only a small subset of [operators](https://github.com/onnx/onnx/blob/master/docs/Operators.md)  are used and verified. So for the rest operators, we may need another task:

- [x] Design the unit test framework to test every operator's conversion (#29 @kuke)

Hints:
- Choose one task every time and assign yourself when you are really ready for working on it; 
- Please append the model to the **supported models** section in README like #19 after its conversion being validated.

**Comment by pkuyym** [comment]
@kuke I think we can collect all the ops needed to be converted required by the listed models. 
It's better to do ` Design the unit test framework to test every operator's conversion` first and we can implement converter for fluid ops parallel. When all operator converter finished, I think it would be easy to validate the models.

**Comment by sidgoyal78** [comment]
Yeah, I agree with @pkuyym . I think it might be better to do design the unit test framework for a variety of operators.

**Comment by kuke** [comment]
I also agree with that  unit test is very important, so I assign it to myself.  But if you are interested in verifying the correctness of models' conversion, you can work on one now. From my experience, a runnable model will help you test the related operator's conversion and the unit test is not necessary.

**Comment by varunarora** [comment]
I'll take READMEs/installation, etc. Thanks @kuke and @pkuyym for your work on the models.

**Comment by kuke** [comment]
@varunarora Great! It is the time to have a friendly README and installation. I am going to continue to work on models' conversion. I think rest of us can pick up a model or general operators' conversion. You see we have a lot of operators waiting to be implemented the conversion.
